src/styles/less/TrophyRoom.less

```diff
@@ -1,15 +1,45 @@
 @import '../less/variables.less';
 
```

Comment:
TrophyRoom.less is the main file I worked in - it contains the CSS code for the trophyroom and leaderboard table.